src/few/summation/interpolatedmodesum.py

```diff
@@ -364,6 +366,10 @@ def sum(
             self.xp.asarray(phase_interp_coeffs), [2, 0, 1]
         ).flatten()
 
+        # for ylm with negative m, need to multiply by (-1)**l as this is assumed to have happened by the kernel
+        ylms = ylms.copy()
+        ylms[num_teuk_modes:] *= (-1) ** l_arr
```

Comment:
Same comments as before, why the $(-1)^l$ here?

Reply:
Needed due to assumptions of the CUDA kernel.